kinder/pkg/test/workflow/taskCmdRunner.go

```diff
@@ -155,8 +172,8 @@ func (c *taskCmdRunner) Run(t *taskCmd, artifacts string, verbose bool) error {
 		cleanup(t.Cmd)
 
 		// otherwise record test case failure and exits with error
-		return c.registerTestCase(t.Name,
-			withFailure(err.Error()),
+		return c.registerTestCase(t.Name, err,
```

Comment:
might be we can also check if some sort of nested error?

Reply:
you mean checking if an error is wrapped using the errors.Wrap calls?